File: corpus/cs2/0011-aprovado-ou-reprovado/solutions/a-optimal/solution.cpp
#include <iostream>

using namespace std;

int main() {
    double a, b;
    if (cin >> a >> b) {
        double media = (a + b) / 2.0;
        
        if (media >= 7.0) {
            cout << "Aprovado\n";
        } else if (media >= 4.0) {
            cout << "Recuperacao\n";
        } else {
            cout << "Reprovado\n";
        }
    }
    return 0;
}

File: corpus/cs2/0011-aprovado-ou-reprovado/solutions/a-optimal/solution.py
import sys

def main():
    data = sys.stdin.read().split()
    if len(data) >= 2:
        a = float(data[0])
        b = float(data[1])
        media = (a + b) / 2.0
        
        if media >= 7.0:
            print("Aprovado")
        elif media >= 4.0:
            print("Recuperacao")
        else:
            print("Reprovado")

if __name__ == "__main__":
    main()
